### PR TITLE
fixed some doc

### PR DESCRIPTION
**Pull request by timea-solid**
Internal documentation was not displaying correctly. I also reworded some sentences. 
Now we have a link to releases from the README under the Changelog.

### CODE REVIEW

File: src/site/apt/data-modeling/multiple-resources.apt.vm [hunk 1]
```diff
@@ -22,7 +24,7 @@ public class MyApplication {
         Session session = OpenIdSession.ofClientCredentials(issuer, clientId, clientSecret, "client_secret_basic");
         SolidSyncClient client = SolidSyncClient.getClient().session(session);
 
-        URI uri = URI.create("https://storage.example/booklibrary/");
+        URI uri = URI.create("https://storage.example/BookLgitibrary/");
```
Comment (acoburn) [inline src/site/apt/data-modeling/multiple-resources.apt.vm:27]: Is this change intended?

**Review by acoburn — APPROVED** [review]

File: src/site/apt/data-modeling/multiple-resources.apt.vm (hunk 1 above)
Comment (timea-solid) [inline src/site/apt/data-modeling/multiple-resources.apt.vm:27]:
```suggestion
        URI uri = URI.create("https://storage.example/BookLibrary/");
```

Comment (timea-solid) [reply src/site/apt/data-modeling/multiple-resources.apt.vm]: definitely not. 😅
